src/platform/plugins/shared/data/common/search/search_source/fetch/get_search_params.ts

```diff
@@ -35,10 +35,13 @@ export function getSearchParamsFromRequest(
   const searchParams = { preference: getEsPreference(getConfig) };
   const dataView = typeof searchRequest.index !== 'string' ? searchRequest.index : undefined;
   const index = dataView?.title ?? `${searchRequest.index}`;
+  const trackTotalHits = searchRequest.track_total_hits ?? searchRequest.body.track_total_hits;
 
   // @ts-expect-error elasticsearch@9.0.0 `query` types don't match (it seems like it's been wrong here for a while)
   return {
     ...searchRequest,
+    ...(typeof trackTotalHits !== 'undefined' && { track_total_hits: trackTotalHits }),
+    track_total_hits: trackTotalHits,
```

Comment:
Thx for fixing this so quickly 🏎️  qq: there seems to be redundancy here, isn't this sufficient?

```suggestion
    track_total_hits: trackTotalHits,
```

or even 

```suggestion
    track_total_hits,
```

does it do harm when it's undefined?

Reply:
Oops, yeah you're right. Fixed in [d35b122](https://github.com/elastic/kibana/pull/215245/commits/d35b1225a3bca80a212db93d58fdf84a63c772df)